packages/react/src/components/Datepicker/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import 'react-day-picker/lib/style.css';
+import Icon from '../Icon';
+import IconButton from '../IconButton';
+
+const DatepickerNavigation = React.forwardRef(
+  ({ month, onPreviousClick, onNextClick }, ref) => {
+    return (
+      <div ref={ref} className='f-Datepicker--navbar'>
+        {month.toLocaleString('default', { month: 'long' })}{' '}
+        {month.getFullYear()}
+        <div className='f-Datepicker--actions'>
+          <IconButton
+            onClick={() => onPreviousClick()}
+            icon={Icon.ICONS.IconChevronLeft}
+          />
+          <IconButton
+            onClick={() => onNextClick()}
+            icon={Icon.ICONS.IconChevronRight}
+          />
+        </div>
+      </div>
+    );
+  },
+);
+
+const DatepickerInput = React.forwardRef(
+  ({ onClick, placeholder, value, isInvalid }, ref) => (
+    <div
```

Comment:
Interesting, why not using an Input?

Reply:
For the design that I was implementing this component is just for display, it's not editable